usaspending_api/transactions/tests/integration/test_assistance_etl.py

```diff
@@ -251,3 +251,165 @@ def test_data_transfer_from_broker(load_broker_data):
+VALUES
+    (E'2017-09-16 22:22:42.760993','{insert_now_time}',{new_published_fabs_id},E'07/12/2017',E'C',E'06',E'UNKNOWN TITLE',E'Columbus State Community College',NULL,E'091',NULL,E'9100',E'3',E'NON',E'06',E'84.033',E'C',0,E'P033A173267',520000,NULL,E'091',NULL,NULL,E'550 E Spring St',NULL,NULL,E'USA',NULL,NULL,NULL,E'43215',E'1722',0,0,E'08/31/2023',NULL,E'OH18000',E'03',E'USA',NULL,E'432151722',2,NULL,NULL,E'03',E'520000.0',E'Federal Work-Study Program',E'Department of Education (ED)',E'Department of Education',E'EDUCATION, DEPARTMENT OF (9100)',NULL,TRUE,E'Delaware',E'Ohio',E'COLUMBUS',E'Columbus',E'049',E'Franklin',E'OH',E'Ohio',E'2017-07-21 00:00:00','{case_change_afa}',TRUE,NULL,NULL,NULL,NULL,E'UNITED STATES',E'UNITED STATES',E'041',NULL,E'OH',E'43215',E'1722',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,E'ASST_NON_P033A173267_9100',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,E'Single ZIP Code',E'awardee-uei',E'parent-uei',E'funding-opportunity-goals',E'funding-opportunity-number',123456)
+    ;
```

Comment:
I think this might be easier to follow if you created a dict of for the record outside of the string and then inside did something like this:
```
INSERT INTO "{BROKER_TABLE}"
({", ".join(record.keys())})
VALUES
    ({", ".join(record.values())})
```

Reply:
True, but mostly I was just copying-pasting the pattern from [the prior test that exists and its data fixture](https://github.com/fedspendingtransparency/usaspending-api/blob/92fc1be561284dbe25ee7cbf2be94e5f9b2d96e3/usaspending_api/transactions/tests/integration/test_assistance_etl.py#L15-L121) and didn't want to get bogged down in optimizing both tests (if I do it for the new one, I should refactor the old one).

Reply:
Fine.